Software: Socket.io (#21)
PoCInnovation/Workshops: software/README.md

- Create two consumer services
- Publish messages into Kafka

##  19 - [Create Chat multi-client with Socket.IO](./1.Socket.io)

> Learn Socket.IO with implementation of your own chat online.
> You will :
> - learn basics of Typescript
> - understand API
> - Understand the sockets
> - Understand how transmission data works

Reviewer: ```suggestion
Learn Socket.IO by implementing your own online chat.
You will :
- Learn Typescript fundamental
- Work with socket and http servers
- Understand sockets
- Understand how transmits data
```
